src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String fruitName;
+    private int quantity;
```

Comment:
The 'quantity' attribute seems to be a part of the business logic rather than the Fruit model itself. Typically, a Fruit class would represent a type of fruit without holding its quantity. Consider moving the quantity management to a service or storage class that handles the business logic.